doc/fluidsettings.xml

```diff
@@ -240,6 +240,14 @@ https://stackoverflow.com/a/6251757
             </desc>
         </setting>
         <setting>
+            <name>alsa.autoconnect</name>
+            <type>bool</type>
+            <def>0 (FALSE)</def>
+            <desc>
+                If 1 (TRUE), automatically connects FluidSynth to available ALSA MIDI input ports.
```

Comment:
@tomcucinotta Haven't tested it yet. You could let me know if this documentation is sufficient.

Reply:
docs seem ok, thanks!

Reply:
Hi, I also just recompiled this new alsa-autocon branch, and tried the feature, plugging my USB MIDI adapter, then launching:

  ./src/fluidsynth -o audio.alsa.autoconnect=1 -a alsa -i /usr/share/sounds/sf2/FluidR3_GM.sf2 -s

I've got the USB MIDI adapter, as well as the MIDI through, readily connected to FLUID Synth :-)!

Reply:
Confirmed.